MSAL/src/native_auth/controllers/sign_in/MSALNativeAuthSignInControlling.swift

```diff
@@ -25,17 +25,17 @@
 import Foundation
 
 protocol MSALNativeAuthSignInControlling {
-    typealias SignInPasswordControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInPasswordStartResult>
-    typealias SignInCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInStartResult>
+
+    typealias SignInControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInStartResult>
     typealias SignInAfterPreviousFlowControllerResponse =
         MSALNativeAuthControllerTelemetryWrapper<Result<MSALNativeAuthUserAccountResult, MSALNativeAuthError>>
     typealias SignInSubmitCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInVerifyCodeResult>
     typealias SignInSubmitPasswordControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInPasswordRequiredResult>
     typealias SignInResendCodeControllerResponse = MSALNativeAuthControllerTelemetryWrapper<SignInResendCodeResult>
 
-    func signIn(params: MSALNativeAuthSignInWithPasswordParameters) async -> SignInPasswordControllerResponse
+    func signIn(params: MSALNativeAuthSignInWithPasswordParameters) async -> SignInControllerResponse
```

Comment:
do you think would be possible to use only one param class in this case? So, we can have only one signIn method

Reply:
Updated, signUp, signIn code and password methods unified